mmv1/third_party/terraform/services/compute/resource_compute_instance_template.go.erb

```diff
@@ -1350,10 +1372,9 @@ func resourceComputeInstanceTemplateCreate(d *schema.ResourceData, meta interfac
 		ReservationAffinity:        reservationAffinity,
 	}
 
-	if _, ok := d.GetOk("effective_labels"); ok {
-		instanceProperties.Labels = tpgresource.ExpandEffectiveLabels(d)
```

Comment:
`effective_labels` is introduced in Terraform Google Provider 5.0 and should stay here.

https://registry.terraform.io/providers/hashicorp/google/latest/docs/guides/version_5_upgrade#provider